tasks/check-data-keys/README.md

```diff
@@ -15,6 +15,9 @@ Currently, `releaseNotes`, and `cdn` are the only supported systems.
 | systems  | The systems to check that all data keys are present for | Yes      | []            |
 | schema   | The URl to the schema                                   | Yes      | https://raw.githubusercontent.com/konflux-ci/release-service-catalog/refs/heads/development/schema/dataKeys.json |
 
+## Changes in 1.0.2
+* Strip the `.git/` suffix from the `schema` value if present. Some `taskGitUrl`s add this suffix which results in invalid schema files being passed
```

Comment:
Where is this coming from? Do we know? How come it's just some taskGitUrls and not all?

Reply:
We don't pass it in e2e https://github.com/konflux-ci/e2e-tests/blob/main/pkg/constants/constants.go#L137 which is why it passed. We do as the pipeline default https://github.com/konflux-ci/release-service-catalog/blob/development/pipelines/managed/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml#L50 which is why a user hit it. I *think* this is how this happened

Reply:
I see, turns out konflux-release-data has `https://github.com/konflux-ci/release-service-catalog.git` everywhere. And it's also what tekton has as an example in the documentation: https://tekton.dev/docs/pipelines/git-resolver/

So it would probably make sense to use that in e2e as well.

Reply:
Fix: https://github.com/konflux-ci/e2e-tests/pull/1480

Reply:
thank you